user-guide/python.rst

```diff
@@ -270,225 +240,129 @@ Please click on the link indicated to see examples of how to use the `PyTorch an
 More detail on the Cirrus GPU nodes can be found at https://cirrus.readthedocs.io/en/main/user-guide/gpu.html .
 
 
-Custom Miniconda3 Environments
-------------------------------
+Extending a centrally-installed Miniconda3 environment
```

Comment:
I would maybe call that title something more explicit like `Installing packages locally` because that's what users will be looking for when skimming through the page.

Reply:
Done